files

```diff
@@ -0,0 +1 @@
+templates
```

Comment:
I'm a bit confused what this file is for?

Reply:
This is how git handles symlinks. Saddly OpenLDAP only looks in the files folder inside the playbook-dir. to avoid creating another set of directories i symlinked them there.